simpeg/directives/directives.py

```diff
@@ -3426,3 +3426,70 @@ def endIter(self):
             for directive in directiveList:
                 if not isinstance(directive, SaveIterationsGeoH5):
                     directive.endIter()
+
+
+class ScaleMisfitMultipliers(InversionDirective):
+    """
+    Scale the misfits by a factor.
```

Comment:
Can you describe what it is you're scaling the misfit by?  This is only useful for joint?  If so should that be part of the name of the directive?